db/migrate/20200107154252_correct_dns_timeout_setting.rb

```diff
@@ -0,0 +1,7 @@
+class CorrectDnsTimeoutSetting < ActiveRecord::Migration[5.2]
+  def up
+    unless Setting[:dns_timeout].is_a?(Integer) || Setting[:dns_timeout].is_a?(Array)
+      Setting.find_by_name('dns_timeout').update_column(:value, nil)
```

Comment:
Looks like this fails if the settings haven't been initialized yet, let's be defensive:
```suggestion
      Setting.find_by_name('dns_timeout')&.update_column(:value, nil)
```

Reply:
*Facepalm* rebased.